samples/Calling/src/app/views/HomeScreen.tsx

```diff
@@ -145,15 +180,29 @@ export const HomeScreen = (props: HomeScreenProps): JSX.Element => {
   );
 };
 
-/* @conditional-compile-remove(PSTN-calls) */
+/* @conditional-compile-remove(1-n-calling)  */
 /**
  * splits the participant Id's so we can call multiple people.
- *
  */
-const parseParticipants = (participantsString?: string): string[] | undefined => {
+const parseAcsParticipants = (participantsString?: string): string[] | undefined => {
   if (participantsString) {
     return participantsString.replace(' ', '').split(',');
   } else {
     return undefined;
   }
 };
+
+/* @conditional-compile-remove(PSTN-calls) */
+/**
+ * Parse the unformatted string that is given from the dialpad to something that the
+ * Calling adapter will accept as a PSTN participant.
+ *
+ * @param participant - string stored from dialpad inputs.
+ * @returns formatted phone number to be added to call locator.
+ */
+const parseDialPadParticipant = (participant?: string): string[] | undefined => {
```

Comment:
This should ideally be happening in the dialpad itself.
I can't think of a scenario where a formatted string returned by the dialpad is useful.